crates/sui-core/src/authority_active/gossip/mod.rs

```diff
@@ -428,7 +445,7 @@ where
                             // batch has been fully processed.
                             last_seq_in_cur_batch = seq;
 
-                            let fut = handler.handle_digest(self, digests);
+                            let fut = handler.handle_digest(self, digests).await?;
```

Comment:
As discussed above. All the work, including blocking will happen here in the call site / task calling handle_digest. That negates the point of then executing things in parallel in futures ordered is it not?

Reply:
You are correct - I was just trying to match the API of the NodeSyncDigestHandler but I went too far.

Reply:
Ok, should be better now